auslib/test/test_db.py

```diff
@@ -460,6 +460,7 @@ def testHistoryTableHasAllColumns(self):
         self.assertTrue('foo' in columns)
         self.assertTrue('changed_by' in columns)
         self.assertTrue('timestamp' in columns)
+        self.assertTrue('data_version' in columns)
```

Comment:
I noticed that this columns were missing from these tests while trying to write a useful test for this. I couldn't think of any useful tests to add, but we may as well keep this...